src/core/sig_state.ml

```diff
@@ -196,6 +194,19 @@ let open_sign : sig_state -> Sign.t -> sig_state = fun ss sign ->
   in
   {ss with in_scope; builtins; unops; binops; pp_hints}
 
+(** [of_sign sign] creates a state from the signature [sign] with ghost
+    signatures opened. *)
+let of_sign : Sign.t -> sig_state = fun sign ->
+  let empty =
+    { signature = sign; in_scope = StrMap.empty; aliases = StrMap.empty
```

Comment:
You can use dummy now.

Reply:
Where? This function acts as the constructor of `sig_state` doesn't it? We don't have a dummy `sig_state` elsewhere.

Reply:
Yes, see comment above.